src/nvim/testdir/test_usercommands.vim

```diff
@@ -0,0 +1,44 @@
+" Tests for user defined commands
+
+" Test for <mods> in user defined commands
+function Test_cmdmods()
+  let mods = ''
+
+  command! -nargs=* MyCmd let mods .= '<mods> '
+
+  MyCmd
+  aboveleft MyCmd
+  belowright MyCmd
+  botright MyCmd
+  browse MyCmd
+  confirm MyCmd
+  hide MyCmd
+  keepalt MyCmd
+  keepmarks MyCmd
+  keepjumps MyCmd
+  keeppatterns MyCmd
+  lockmarks MyCmd
+  noswapfile MyCmd
+  silent MyCmd
+  tab MyCmd
+  topleft MyCmd
+  verbose MyCmd
+  vertical MyCmd
+  aboveleft belowright botright browse confirm hide keepalt keepmarks
+        \ keepjumps keeppatterns lockmarks noswapfile silent tab
+        \ topleft verbose vertical MyCmd
+  call assert_equal(' aboveleft belowright botright browse confirm ' .
+        \ 'hide keepalt keepmarks keepjumps keeppatterns lockmarks ' .
+        \ 'noswapfile silent tab topleft verbose vertical aboveleft ' .
+        \ 'belowright botright browse confirm hide keepalt keepmarks ' .
+        \ 'keepjumps keeppatterns lockmarks noswapfile silent tab topleft ' .
+        \ 'verbose vertical ', mods)
+  let mods = ''
```

Comment:
original patch calls this `g:mods`, and that's part of our style guidelines anyways.